script.js

```diff
@@ -0,0 +1,221 @@
+// User starts with either X or O
+// User clicks on a space, and that space shows an X or an O
+  // space displays the X or O, depending on which one was currently in effect
+  // turn switches to other player
+// Play continues until one of two things happens:
+    // No spaces remain
+    // A player, either X or O, has won
+// Reset button at the bottom of the page clears all entries
+
+var squareOne = $("#topRow .left");
+var squareTwo = $("#topRow .middle");
+var squareThree = $("#topRow .right");
+var squareFour = $("#middleRow .left");
+var squareFive = $("#middleRow .middle");
+var squareSix = $("#middleRow .right");
+var squareSeven = $("#bottomRow .left");
+var squareEight = $("#bottomRow .middle");
+var squareNine = $("#bottomRow .right");
+var resetButton = $("button");
+
```

Comment:
I think this first bit of code gets your into some trouble. Instead why not give all elements the same class and label them all.

Here's a thought experiment. Lets say all the td's have class `square` and each one would have a unique word to that square in that td. Then I wrote this code:

```js
$(".square").on("click", function(){
  console.log($(this).html())
}
``` 

What would be logged as I clicked each unique word? 

In this way, we no longer have to be specific to each square.

That being said, everything else would need to get largely revamped to make this code work, but this would be a starting point.